Dockerfile

```diff
@@ -11,12 +11,18 @@ ENV ADMIN_PORT=3005
 EXPOSE ${HTTP_PORT}/tcp ${WS_PORT}/tcp ${CDN_PORT}/tcp ${RTC_PORT}/tcp ${ADMIN_PORT}/tcp
 
 # install required apps
-RUN apk add --no-cache --update git python3 py-pip make build-base
+RUN apk add --no-cache --update git python3 py-pip make build-base g++ pkgconfig pixman-dev cairo-dev pango-dev
 RUN ln -s /usr/bin/python3 /usr/bin/python
+RUN npm install --global patch-package --save
 
 # Run as non-root user
 # RUN adduser -D fosscord
 # USER fosscord
 
 WORKDIR /srv/fosscord-server/bundle
-ENTRYPOINT ["npm", "run", "start:bundle"]
\ No newline at end of file
+COPY package.json ./
+RUN npm install --omit optional
+COPY . .
+RUN patch-package
+RUN npm run build
+CMD ["npm", "run", "start:bundle"]
```

Comment:
Unnecessary amount of commands (each RUN/COPY adds a layer to the final image). It's way better if you organize your `COPY` commands (to use only one instead of two) and run all commands in a single `RUN` statement.

Reply:
you can run all those commands by just running `npm run setup` btw